stories/Form Elements/Fieldset.stories.tsx

```diff
@@ -104,9 +104,18 @@ export const WithCustomLegendSizeL: Story = {
 
 export const WithCustomLegendSizeXL: Story = {
   name: 'With Custom Size (XL)',
-  render: (args) => (
+  render: () => (
     <Fieldset>
       <Fieldset.Legend size="xl">What is your address?</Fieldset.Legend>
     </Fieldset>
   ),
 };
+
+export const WithHTML: Story = {
```

Comment:
Do you mind placing this after `WithLegendAsAPageHeading`?

Then we can name it `WithLegendHTML` so it's not stuck below the custom sizes

```suggestion
export const WithLegendHTML: Story = {
```